engines/e_capi.c

```diff
@@ -1466,6 +1503,7 @@ static CAPI_KEY *capi_get_key(CAPI_CTX * ctx, const TCHAR *contname,
 {
     DWORD dwFlags = 0;
     CAPI_KEY *key = OPENSSL_malloc(sizeof(*key));
```

Comment:
Call OPENSSL_zalloc, or add one more lien of code :
 key->id = NULL
Else, it will crash ajust after any first "goto err:" call .